### ISSUE TITLE
Connectivity Package

### ISSUE THREAD

**Issue by robbisg** [robbisg/pyitab#60]
This is to collect issues related to connectivity package:
-----
*Related issues:*
- [ ] Load of gm, wm etc #20 
- [ ] Load of feature attributes (movement) #21
- [ ] Regressor Transformer #22
- [ ] Frame Displacement Transformer #23
- [ ] Filter #24
- [ ] PCA/Average/MVPC Transformer #25
- [ ] Correlation / Coherency Analyzer #26

In general the connectivity is a ```Transformer``` because it maps a dataset in source/voxels space to connectivity matrix space.
So basically each class that performs a mapping should implement ```transform``` instead of ```fit```.
They transform with a metric or a method.

TODO:
- [ ] ```SlidingWindowEstimator```
- [ ] How to cope with MNE classes?
